### PR TITLE
Improve the flip restriction

### PR DESCRIPTION
**Pull request by thegrb93**
## About The Pull Request
You still get the benefit of spending 8 quirk points, but now anyone can flip so long as they are fast enough which is affected by hunger, injury, missing limbs, etc, and meth counteracts this and will let you flip anyway.

## Why It's Good For The Game
You don't have to gimp your character with 8 points of quirks just to have an emote

## Changelog

:cl:
balance: The Freerunning quirk is no longer required to *flip
/:cl:

### CODE REVIEW

**Comment by thegrb93** [comment]
[2024-10-20 22-51-21.webm](https://github.com/user-attachments/assets/7d1c5e9b-0b55-457c-8206-e4b0c66c2910)

**Comment by Swiftfeather** [comment]
> 2024-10-20.22-51-21.webm

bonk bonk bonk bonk bonk

**Comment by sippykot** [comment]
amazing 🙏

File: modular_skyrat/modules/emotes/code/emotes.dm [hunk 1]
```diff
@@ -78,11 +78,12 @@
 				carbon_user.next_smell = world.time + SMELL_COOLDOWN
 			current_turf.pollution.smell_act(user)
 
-/datum/emote/flip/can_run_emote(mob/user, status_check, intentional)
+// BUBBER EDIT: Improved flip emote checks
```
Comment (Arturlang) [inline modular_skyrat/modules/emotes/code/emotes.dm:81]: Just delete it entirely, we're no longer modular not pulling with skyrat

**Comment by Arturlang** [comment]
Requires discussion about stamina costs and requirements for flipping

**Comment by Arturlang** [comment]
Alrighty, so there should be a stamina requirement for flipping, and each flip should take some amount of stamina @thegrb93

**Comment by thegrb93** [comment]
> Alrighty, so there should be a stamina requirement for flipping, and each flip should take some amount of stamina @thegrb93

The stun and flop on the floor if you do it too much isn't enough to discourage spamming it? How much stamina?

**Comment by Swiftfeather** [comment]
> Alrighty, so there should be a stamina requirement for flipping, and each flip should take some amount of stamina @thegrb93

As long as freerunning still lets me go speeeeeEEEeeeEeennn

**Comment by thegrb93** [comment]
[2024-11-09 14-22-33.webm](https://github.com/user-attachments/assets/a837e4a4-3363-45e0-bbf9-bf5e07701aa1)

**Comment by FridayRemada** [comment]
over-flipping as a synth causes you to throw up, draining your battery by causing 'starvation' even though synths don't eat to subsist.

**Pull request closed without merging**